test/test_ITF1788/run_ITF1788.jl

```diff
@@ -14,8 +14,9 @@ include("libieeep1788_reduction.jl")
 include("libieeep1788_set.jl")
 include("mpfi.jl")
 
-using IntervalContractors
-include("abs_rev.jl")
-include("libieeep1788_rev.jl")
-include("libieeep1788_mul_rev.jl")
-include("pow_rev.jl")
+# TODO: CHECK CONTRACTORS
```

Comment:
I think the issue here is that `IntervalContractors.jl` uses certain functionality that has changed in 1.0-dev branch.